scripts/contract-upgrade-governance.sh

```diff
@@ -190,6 +190,13 @@ case "$chain_name" in
     explorer="https://optimistic.etherscan.io/address/"
     evm=true
     ;;
+  base)
+    echo "Need to specify the base explorer URL!"
+    exit 1  
+    chain=30
+    explorer="??/address/"
```

Comment:
https://goerli.basescan.org/

Reply:
Don't think we want testnet here. This is a reminder to add the mainnet one when we know it.

Reply:
ohhh. 👍